server/src/main/java/org/elasticsearch/cluster/routing/allocation/decider/DiskThresholdDecider.java

```diff
@@ -66,11 +68,22 @@
 public class DiskThresholdDecider extends AllocationDecider {
 
     private static final Logger logger = LogManager.getLogger(DiskThresholdDecider.class);
+    private static final DeprecationLogger deprecationLogger = DeprecationLogger.getLogger(DiskThresholdDecider.class);
 
     public static final String NAME = "disk_threshold";
 
     public static final Setting<Boolean> ENABLE_FOR_SINGLE_DATA_NODE =
```

Comment:
I think this means the `enableForSingleDataNode` field is now always true and so can be inlined.